android/libraries/rib-base-test/src/main/java/com/badoo/common/ribs/InteractorTestHelper.kt

```diff
@@ -21,17 +23,14 @@ private val buildParams = BuildParams.Empty()
 
 class InteractorTestHelper<View : RibView>(
     val interactor: Interactor<*, View>,
-    val viewFactory: ((ViewGroup) -> View?)? = null,
-    router: Router<*, *, *, *, View>? = null
```

Comment:
Is there any cases when you need to mock router? or validate something?

Reply:
Not in Interactor anymore. Interactor shouldn't need to do anything directly to Router now.